src/inverseRobot.js

```diff
@@ -8,6 +8,23 @@
 
 function inverseRobot(robot) {
   // write code here
+  const invertedRobot = {};
+  const valuesSeen = {};
```

Comment:
The 'valuesSeen' object is redundant. You only need to check if the 'value' already exists as a key in the 'invertedRobot' object. This will simplify the code and reduce memory usage.